pages/docs/installation/installation-overview.md

```diff
@@ -23,15 +23,15 @@ To find the right method for you, follow this decision graph, or simply read on!
 
 ### Operating systems
 
-**Linux**
+#### Linux
```

Comment:
How do such changes interfere with the TOC?

Style-wise `**...**` sometimes just looks better.

Reply:
They will appear in the TOC as level 4 headings. My motivation was that this was triggering the linter, as it is essentially a heading of hard-coded style. If this is an intended styling workaround, I can add exceptions, but we should fix it. Shall I just do this?

Here is how it looks like:

![Screenshot_2021-04-23 Installing preCICE preCICE - The Coupling Library](https://user-images.githubusercontent.com/4943683/115893422-714ed080-a458-11eb-98f8-fdb1533d326f.png)

Reply:
I suppressed this for now, reverting the previous version.